app/grandchallenge/components/tasks.py

```diff
@@ -632,8 +635,16 @@ def provision_job(
     )
     executor = job.get_executor(backend=backend)
 
+    if not job.inputs_complete:
+        logger.warning("Nothing to do: inputs are not complete.")
```

Comment:
We expect this state often right? Nothing special is going wrong or needs to be highlighted:

```suggestion
        logger.info("Nothing to do: inputs are not complete.")
```

Reply:
True. Info logging level makes more sense.